<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinningParametersDialog</class>
 <widget class="QDialog" name="BinningParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>249</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>249</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>249</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Параметры бина</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/imgs/3d.png</normaloff>:/icons/imgs/3d.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="x0Label">
     <property name="text">
      <string>Xo=</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="x0LineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="y0Label">
     <property name="text">
      <string>Yo=</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="y0LineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dXLabel">
     <property name="text">
      <string>dX=</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="dXLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dYLabel">
     <property name="text">
      <string>dY=</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="dYLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="i0Label">
     <property name="text">
      <string>Io=</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="i0LineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="j0Label">
     <property name="text">
      <string>Jo=</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="j0LineEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="azXLabel">
     <property name="text">
      <string>AzX=</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="azXLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="azYLabel">
     <property name="text">
      <string>AzY=</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="azYLineEdit"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BinningParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BinningParametersDialog</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveSettings()</slot>
 </slots>
</ui>
